test/e2e/runner/main.go

```diff
@@ -13,9 +14,9 @@ import (
 	e2e "github.com/tendermint/tendermint/test/e2e/pkg"
 )
 
-var (
-	logger = log.MustNewDefaultLogger(log.LogFormatPlain, log.LogLevelInfo, false)
-)
+const randomSeed int64 = 2308084734268
```

Comment:
Optional nit: We don't need to state a type, constants infer type from usage.

```suggestion
const randomSeed = 2308084734268
```